runtime/sys.c

```diff
@@ -219,7 +219,7 @@ CAMLprim value caml_sys_exit(value retcode)
 #endif
 #endif
 
-const static int sys_open_flags[] = {
+static const int sys_open_flags[] = {
```

Comment:
Unrelated change.

Reply:
This fixes the `-Wold-style-declaration` warning that I couldn't enable globally, as it's specific to GCC.